src/sbt-test/sbt-typesafe-conductr/sbt-typesafe-conductr-tester/version.properties

```diff
@@ -1 +0,0 @@
-../../../../version.properties
```

Comment:
I was not aware this is possible in scripted tests:

```
addSbtPlugin("com.typesafe.conductr" % "sbt-typesafe-conductr" % sys.props("project.version"))
```

Would be great if this would have been pointed out during review, would have learnt something about scripted right away :)